components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt

```diff
@@ -159,7 +159,7 @@ class MozillaSocorroService(
             miniDumpFilePath = crash.minidumpPath,
             extrasFilePath = crash.extrasPath,
             isNativeCodeCrash = true,
-            isFatalCrash = crash.isFatal,
+            isFatalCrash = crash.processType == Crash.NativeCodeCrash.PROCESS_TYPE_MAIN,
```

Comment:
Shouldn't we pass `processType` in and send it through `sendCrashData`?

Reply:
Do we need any Socorro changes to send a new property?

Should we stop sending `isFatal` and send `processType` instead? Or do both?

Reply:
Yes, we'll need Socorro side change after this to show the new property.  I think we should send both, this way we can still filter for fatal/non-fatal crashes and also (after the server side change) see what type of crash it is.